HPHBhelper/hogwarts/models.py

```diff
@@ -0,0 +1,37 @@
+from django.db import models
+
+# Create your models here.
+class Games(models.Model):
+    shortcode = models.CharField(max_length=4)
+    isActive = models.BooleanField(default=False)
+    host = models.ForeignKey(Players)
+
+    class TurnOrder(models.Model):
+        #The turn order is set after a game is created but before it starts
+        pass
+
+
+class Players(models.Model):
```

Comment:
did you run `python manage.py makemigrations`? I don't see the generated migration files in this PR? these should be committed to version control